src/components/table/examples/table-layout-fitDataStretch.tsx

```diff
@@ -0,0 +1,43 @@
+import { Component, h, State } from '@stencil/core';
+import { Column } from '../table.types';
+import { invoices, Invoice } from './invoices';
+
+/**
+ *
+ * ````tsx
+ * layout="fitDataStretch"
+ * ````
+ *
+ * Works just like `fitDataFill`, but unlike the default layout
+ * which resulted in having an empty last column, in this layout
+ * the last existing column will stretch out to fill up the remaining table width.
```

Comment:
In the full-screen view, the last column has only the header:
<img width="1332" alt="Screenshot 2022-09-06 at 06 40 01" src="https://user-images.githubusercontent.com/50618208/188548614-e284671a-12a8-4c95-8a80-06a985397769.png">